helmfile/charts/prometheus-alerts/templates/alerts/daily-checks.yaml

```
@@ -17,15 +18,29 @@ spec:
   groups:
   - name: daily-checks
     rules:
-    - alert: S3BucketContentOver{{.Values.s3BucketPercentLimit}}Percent
+{{- if .Values.s3BucketAlerts.limit.enabled }}
+    - alert: S3BucketContentOver{{.Values.s3BucketAlerts.limit.percent}}Percent
```

I think this is a bit more descriptive
```suggestion
    - alert: S3BucketSizeOver{{.Values.s3BucketAlerts.limit.percent}}Percent
```